### PR TITLE
Update Registry-linter docker file to use a specific version of spectral

### PR DESCRIPTION
**Pull request by giteshk**
Additionally added compiled proto file to Registry server container

### CODE REVIEW

File: containers/registry-server/Dockerfile [hunk 1]
```diff
@@ -43,6 +45,7 @@ RUN apk add --no-cache ca-certificates
 
 # Copy the binary to the production image from the builder stage.
 COPY --from=builder /app/registry-server /registry-server
+COPY --from=builder /app/deployments/envoy/proto.pb /proto.pb
```
Comment (timburks) [inline containers/registry-server/Dockerfile:48]: Why is this needed here? Usually it's an input to envoy, but I don't think there's an envoy instance in this container. (is there?)

Comment (giteshk) [reply containers/registry-server/Dockerfile:48]: This is needed when we test the setup using istio and have a sidecar proxy for grpc transcoding.

Comment (timburks) [reply containers/registry-server/Dockerfile:48]: So do you have the sidecar read this out of the container?

**Review by timburks — APPROVED** [review]

**Review by shrutiparabgoogle — APPROVED** [review]

File: containers/registry-server/Dockerfile (hunk 1 above)
Comment (giteshk) [reply containers/registry-server/Dockerfile:48]: When I look at the file for the hosted solution. The files are copied into an empty volume that is shared between the running registry server pod and the istio filter.
